cookbooks/swift/recipes/configs.rb

```diff
@@ -106,6 +106,13 @@
   creates "/usr/local/share/ca-certificates/extra/saio_ca.crt"
```

Comment:
i think a lot of other "full_reprovision" guys just block out the create `if not full_reprovision` so that the command gets re-run